docs/contributing/how-to-create-new-generated-symbol.md

```diff
@@ -63,30 +67,46 @@ The very basic implementation may be:
             environmentSettings.Host.Logger.LogDebug("[{macro}]: Variable '{var}' was assigned to value '{value}'.", nameof(HelloMacro), config.VariableName, greetings);
         }
 
-        protected override HelloMacroConfig CreateConfig(IGeneratedSymbolConfig generatedSymbolConfig) => new(this, generatedSymbolConfig);
+        public override HelloMacroConfig CreateConfig(IEngineEnvironmentSettings environmentSettings, IGeneratedSymbolConfig generatedSymbolConfig) => new(this, generatedSymbolConfig);
```

Comment:
it would be good to add a basic test for this class

Reply:
done

Reply:
This test should be added to the UnitTests, not to the doc :)

Reply:
FakeMacroTests class is added